cmd/sponged/tests/item_test.go

```diff
@@ -260,5 +260,38 @@ func TestDeleteItem(t *testing.T) {
 			}
 			t.Logf("\t%s\tShould not be able to retrieve the item.", tests.Success)
 		}
+
+		//----------------------------------------------------------------------
+		// Check the inferred relationships.
+
+		opts := map[string]interface{}{
+			"database_name": cfg.MustString("MONGO_DB"),
+			"username":      cfg.MustString("MONGO_USER"),
+			"password":      cfg.MustString("MONGO_PASS"),
+		}
+
+		store, err := cayley.NewGraph("mongo", cfg.MustString("MONGO_HOST"), opts)
+		if err != nil {
+			t.Fatalf("\t%s\tShould be able to connect to the cayley graph : %s", tests.Failed, err)
+		}
```

Comment:
Curious if you could move this `NewGraph` code out into a separate function? I expect it's used more.